templates/partials/page-footer.hbs

```diff
@@ -1,22 +1,32 @@
 <footer role="contentinfo"{{#if footerModifierClass}} class="{{footerModifierClass}}"{{/if}}>
-  <p>made with ❤️ <a href="/"><svg role="img" class="grav-c-logo" aria-label="buildit @ wipro digital" width="300" height="33"><use xlink:href="#buildit-logotype"></use></svg></a></p>
+  <p>made with ❤️ 
+    <a href="/">
+      <title>Buildit Logo</title>      
```

Comment:
shouldn't the title go inside the `<svg>` tag?

Reply:
We do have `<title>`s and `<description>` in our source SVGs so I'm guessing the compressor is removing them for some odd reason, which shouldn't be the case... @james-nash any ideas?

Reply:
@ThePeach @ana-buildt 
So, `<title>` definitely needs to be within the `<svg>`.

We're using `gulp-svg-symbols` to generate our symbol defs. You're right, the source files do have correct `<title>`s in them. Not sure why those aren't being preserved. Let me investigate and get back to you.

Reply:
So, the SVG symbol defs from Gravity _do_ include the `<title>`s (do a view source on the website page to see for yourself, they're all there).

The trouble seems to be that browser's will not expose the `<title>` (or `<description>`) of an SVG being referenced by `<use>`. Therefore, screen readers can't access that info either.

It seems that, wherever you reference an SVG symbol, you need to either include a `<title>` there directly. Or, you can reference the symbol's one (if it has an ID) via an `aria-labelledby` attribute.

See:
* https://css-tricks.com/accessible-svgs/
* https://stackoverflow.com/questions/38353094/making-svg-symbol-accessible

Currently, our SVG titles do not have ID attributes.

I will have a look at `gulp-svg-symbols` in Gravity, to see if that's something we can easily rectify there. Assuming it is, you can use the `aria-labelledby` trick.

Until then, your only option for the website will be to manually add `<title>` attributes. Given that they only occur infrequently, that might be the best for MVP.

Reply:
So, to clarify, where you have something like:
```html
<svg role="img" class="..." width="..." height="...">
    <use xlink:href="#buildit-foo"></use>
</svg>
```

...it should be something like:
```html
<svg role="img" class="..." width="..." height="...">
    <title>A picture of a unicorn pooping rainbows</title>
    <use xlink:href="#buildit-foo"></use>
</svg>
```